std/math.d

```diff
@@ -501,7 +502,7 @@ auto abs(Num)(Num z) @safe pure nothrow @nogc
     return hypot(z.re, z.im);
 }
 
-/** ditto */
+///
```

Comment:
Why this change? Shouldn't the overloads be ditto'ed together into a single doc entry?